TextRPG_Team25/Battle.cs

```diff
@@ -1,7 +1,42 @@
-﻿namespace TextRPG_Team25
+﻿using System.Numerics;
+using System.Threading;
+
+namespace TextRPG_Team25
 {
     internal class Battle
     {
+        public void MonsterAttack()
+        {
+            int battleHp = player.hp; //전투에서쓰는 HP따로 선언
+
+
+        for (int battleEnermy = 0; battleEnermy <= EnermyList.Count; battleEnermy++)  //랜덤으로 생성된 적 개체수만큼 실행
+            {
+
+                int randomDmg = new Random().Next(1, 11) //적 공격력의 오차값 10퍼센트
+
+
+        if (randomDmg == 1)
+                {
+                    if (randomDmg == 1) { double i = Monster.attack; i = i * 90 / 100; double enermyDmg = Math.Round(i); }     //10퍼센트 확률로 90퍼센트의 데미지 반올림
+                    else if (randomDmg == 10)
+                    {
+                        if (randomDmg == 1) { double i = Monster.attack; i = i * 90 / 100; double enermyDmg = Math.Round(i); } //10퍼센트의 확률로 110퍼센트의 데미지 반올림
+
+
+                        if (EnermyList[i].EnermyDead == true) { randomDmg = 0 }
+
+                        Console.WriteLine($"Lv.{Monster.level} {Monster.name} 의 공격!");
```

Comment:
몬스터는 정적 클래스가 아니기 때문에 바로 사용할 수 없습니다! 따라서 객체를 만든 후에 사용하셔야 해요. 현재로서는 어떤 몬스터를 때릴지 지정이 안 되어있는 상태입니다. 그렇기 때문에 더더욱 몬스터 공격을 Monster 클래스 내부의 메서드로 옮겨야겠네요..! 

```suggestion
                        Console.WriteLine($"Lv.{level} {name} 의 공격!");
```